### ISSUE TITLE
Height settings are ignored on New-UDCodeEditor

### ISSUE THREAD

**Issue by awil897** [ironmansoftware/powershell-universal#2525]
### Version

4.0.4

### Severity

Visual

### Steps to Reproduce

When adding a New-UDCodeEditor section, setting the height flag does not actually set the height on the code editor element anymore. 

```
$options = @{
    fontFamily = "monospace"
    lineNumbers = "off"
    readOnly = "true"
}
New-UDCodeEditor -Id 'codeEditor' -Code "Please make a selection to continue" -Height '300' -Options $options
```

### Expected behavior

```console
Prior to 4.02/4.03, this element displayed with the proper height
```

### Actual behavior

```console
After installing the 4.03 update, the code editor is displayed with only a single line (see image attachment). I went directly from 4.01 to 4.03, so I'm not sure whether 4.02/4.03 introduced the issue.
```

### Visuals

![New-UDCodeEditor](https://github.com/ironmansoftware/issues/assets/93953412/ff3e8111-fefa-49bb-8aff-957021c92358)

**Comment by awil897** [comment]
I was worried this was due to the code editor being nested under multiple elements, but I can also reproduce the issue on a brand new dashboard that was created under 4.05 without any other elements present.

```
$Pages = @()
$Pages += New-UDPage -Name 'Dashboard' -Content {
    New-UDCodeEditor -Id 'codeEditor' -Height '300'
}
New-UDApp -Title 'PowerShell Universal' -Pages $Pages
```

![New-UDCodeEditor2](https://github.com/ironmansoftware/issues/assets/93953412/1cf3a5de-fe41-45c4-ab44-af20c0fd4036)

**Comment by Omzig** [comment]
why does it work in live docs, but not in a dashboard?

**Comment by adamdriscoll** [comment]
The code editor looks like it's resizing to the parent size. If the parent has no defined height, then this happens. It's probably why it works on the docs since it has a parent with defined height. 

```
        New-UDElement -Content {
            New-UDCodeEditor
        } -Attributes @{
            height = "500"
        }
        
```

We added some resizing logic that shouldn't be affecting the height like this. Will fix.

**Comment by adamdriscoll** [comment]
Reopened to ensure we get this merged back into 3.9.x as it is happenign there as well.
